.github/workflows/e2e-nvidia-a10g-x4.yml

```diff
@@ -164,6 +164,19 @@ jobs:
         env:
           GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
 
+      - name: Post job results to Slack
+        if: ${{ always() }}
+        id: slack
+        uses: slackapi/slack-github-action@37ebaef184d7626c5f204ab8d3baff4262dd30f0 # v1.27.0
+        with:
+          # Slack channel id, channel name, or user id to post message.
+          # See also: https://api.slack.com/methods/chat.postMessage#channels
+          # You can pass in multiple channels to post to by providing a comma-delimited list of channel IDs.
+          channel-id: 'cli-e2e-results'
+          slack-message: "*e2e-nvidia-a10g-x4* job run result: ${{ job.status }}\n${{ github.event.pull_request.html_url || github.event.head_commit.url }}"
```

Comment:
I am not sure `github.event.pull_request.html_url` or `github.event.head_commit.url` will exist when the cron job runs. For cron jobs, there is no `github.event` payload. 

https://docs.github.com/en/actions/writing-workflows/choosing-when-your-workflow-runs/events-that-trigger-workflows#schedule

I think you may want to use `${{ steps.check_pr.outputs.pr_or_branch }}` since it should always have a PR number or branch name.

Reply:
Good point - updated the overall message